lib/Service/src/SettingsHandler.cpp

```diff
@@ -68,7 +68,7 @@ bool SettingsHandler::loadConfigurationFile(const String& filename)
     StaticJsonDocument<maxBufferSize> doc;
     char                              buffer[maxBufferSize];
 
-    if (0U == m_fileReader.readFile(filename, buffer, maxBufferSize))
+    if (0U == m_fileHandler.readFile(filename, buffer, maxBufferSize))

```

Comment:
At a lot of places the "U" is not available. Here it is of course. But we should use it consistent in many other places!

Reply:
Do you mean that I should remove the "U" in this line, or add it to all places where it is missing?